doctr/datasets/ic03.py

```diff
@@ -53,7 +53,8 @@ def __init__(
         np_dtype = np.float32
 
         # Load xml data
-        tmp_root = os.path.join(self.root, 'SceneTrialTrain' if self.train else 'SceneTrialTest')
+        tmp_root = os.path.join(
+            self.root, 'SceneTrialTrain' if self.train else 'SceneTrialTest') if sha256 else self.root
```

Comment:
Would it be possible to change the monkeypatch of the unittest to avoid doing this? (which doesn"t make a lot of sense for the actual codebase)